docs/fundamentals/read-operations.txt

```diff
@@ -269,6 +271,106 @@ documents.
             Plot: A sci-fi update of the famous 6th Century poem. In a beseiged land, Beowulf must
             battle against the hideous creature Grendel and his vengeance seeking mother.
 
+.. _laravel-sort:
+
+Sort Query Results
+~~~~~~~~~~~~~~~~~~
+
+To order query results based on the values of specified fields, use the ``where()`` method
+followed by the ``orderBy()`` method.
+
+You can set an **ascending** or **descending** sort direction on
+results. By default, the ``orderBy()`` method sets an ascending sort on
+the supplied field name, but you can explicitly specify an ascending
+sort by passing ``'asc'`` as the second parameter. To
+specify a descending sort, pass ``'desc'`` as the second parameter.
+
+If your documents contain duplicate values in a specific field, you can
+handle the tie by specifying additional fields to sort on. This ensures consistent
+results if the additional fields contain unique values.
+
+This example queries for documents in which the value of the ``countries`` field contains
+``'Indonesia'`` and orders results first by an ascending sort on the
+``year`` field, then a descending sort on the ``title`` field.
+
+.. tabs::
+
+   .. tab:: Query Syntax
+      :tabid: query-syntax
+
+      Use the following syntax to specify the query:
+
+      .. code-block:: php
+
+         $movies = Movie::where('countries', 'Indonesia')
```

Comment:
Could you move the examples to a test?

Reply:
I will move the query syntax examples into a test, but I'm not sure how to move the Controller method snippets into include files.

Reply:
Ok, let's keep the controller example here for now.